### DISCUSSION TITLE
Can we use this for business purposes?

### DISCUSSION THREAD

**Discussion by Skatingdragon** [shopware5/shopware#2799] [Ideas]
I just wanted know more about this??

**Comment by mitelg** [comment]
hey @Skatingdragon 

what is your question? :thinking:

**Reply by Skatingdragon** [reply]
Hey, I am, really. I just went through almost 100 softwares and web applications. 
 
Probably I might ask that? Is that your idea to push that towards a startup as a business?.

**Reply by mitelg** [reply]
No, please read here for more information about that repository and the organisation https://github.com/shopware5
